src/plugins/passwd/passwd.c

```diff
@@ -130,26 +203,25 @@ int elektraPasswdGet (Plugin * handle ELEKTRA_UNUSED, KeySet * returned ELEKTRA_
 	else
 		index = UID;
 	struct passwd * pwd;
-#if HAS_FGETPWENT
 	FILE * pwfile = fopen (keyString (parentKey), "r");
 	if (!pwfile)
 	{
 		ELEKTRA_SET_ERRORF (110, parentKey, "Failed to open configuration file %s\n", keyString (parentKey));
 		return -1;
 	}
+#if defined(USE_FGETPWENT)
 	while ((pwd = fgetpwent (pwfile)) != NULL)
+#elif defined(USE_FGETPWENT_LOCAL)
+	while ((pwd = fgetpwent_l (pwfile)) != NULL)
 #else
-	while ((pwd = getpwent ()) != NULL)
+#error Configuration error in CMakeLists.txt. Neither fgetpwent nor getline were provided. Please open an issue at https://issue.libelektra.org.
```

Comment:
How can this be if we have our own build-in version __getpwent_a?

Reply:
because __getpwent_a needs getline. it still should not trigger because the plugin should be removed by cmake. It is just a safety net.